packages/reliability-tests/tests/high-throughput.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { LightNode, Protocols } from "@waku/interfaces";
+import {
+  createDecoder,
+  createEncoder,
+  createLightNode,
+  utf8ToBytes
+} from "@waku/sdk";
+import {
+  delay,
+  shardInfoToPubsubTopics,
+  singleShardInfosToShardInfo,
+  singleShardInfoToPubsubTopic
+} from "@waku/utils";
+import { expect } from "chai";
+
+import {
+  afterEachCustom,
+  beforeEachCustom,
+  makeLogFileName,
+  MessageCollector,
+  ServiceNode,
+  tearDownNodes
+} from "../../tests/src/index.js";
+
+const ContentTopic = "/waku/2/content/test.high-throughput.js";
+
+describe("High Throughput Messaging", function () {
+  const testDurationMs = 20 * 60 * 1000; // 20 minutes
+  this.timeout(testDurationMs * 1.1);
```

Comment:
what is this multiplier: `1.1`?

Reply:
is this specific just for this test? seems like something we should set at a higher-level config if possible/relevant